microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -243,10 +243,26 @@ Collecting child browsing data and sharing it with the family group organizer re
 The family group organizer may stop the data collection from the family safety portal.  For more information about Microsoft family safety features, navigate to [What is a Microsoft family group?](https://support.microsoft.com/help/12413)
 
 
+<!-- ====================================================================== -->
+## Find on Page 
+
+Find on Page allows you to search a web page for important keywords. Open **Settings and more (...)** > **Find on Page**. A search box will appear. Type any keyword or phrase into this search box and the page will highlight all locations on the page where your exact search term appears. You can then navigate through these terms with the **Previous result** and **Next result** buttons. 
+
+To help you find exactly what you're looking for, Microsoft Edge uses a Microsoft cloud service to find additional related matches for your search. This only happens when the **Include related matches** toggle in Find on Page is on.  
+
+If the **Include related matches** toggle is on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token does not contain any user identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms sent to Microsoft are both deleted immediately after processing occurs. No data is stored for any period of time. 
+
+Related matches in Find on Page are not available in **InPrivate** browsing. 
+
+If you want to stop Microsoft Edge from finding related matches in Find on Page, complete the following steps. 
+1.  Open Find on Page
+1.  Toggle off **Include related matches** 
+
+
```

Comment:
```suggestion
By default, the **Find on Page** feature sends data to Microsoft to help find better results, including synonyms, alternate spellings, and answers to questions.

To prevent **Find on Page** from sending data to Microsoft:

1. Select **Settings and more** > **Privacy, search, and services**.

1. Turn off the **Suggest better matches in Find on Page** toggle.
-->

```
@kendradello - Maybe add this section about the Settings page's toggle for the Find on Page feature.